src/router.js

```diff
@@ -1,5 +1,29 @@
+const path = require('path');
+
+const fs = require('fs');
```

Comment:
you don't need it because you need it in the handlers only

Reply:
done